src/adminCore/admin.controller.ts

```diff
@@ -199,4 +199,20 @@ export class DefaultAdminController {
     )
     return response.redirect(urls.changeListUrl(section, metadata))
   }
+
+  @Post(':sectionName/:entityName/action')
```

Comment:
This url is hardcoded as we use generated url from config file in other places

Reply:
I created a ticket for this one : https://trello.com/c/7ebB8EyK/280-aadev-refactor-the-construction-of-urls-in-the-admincontrollerts-file